docker/proto_files/wallet/myservice.proto

```diff
@@ -11,7 +9,6 @@ message LogTruncated {}
 
 message Error {
   oneof error {
-    errors.OtherError other_error = 1;
```

Comment:
That was the point with this usage. With it, the grpc mock is not working. Without it, everything is fine!